src/selectLimit.js

```diff
@@ -0,0 +1,20 @@
+import Dropdown from 'react-bootstrap/Dropdown'
+import DropdownButton from 'react-bootstrap/DropdownButton';
+
+export default function Limit({setPageLimit, pageLimit}) {
```

Comment:
SelectLimit with a capital letter. Components should start with a Capital letter always

Reply:
Renamed file and checked the other files to make sure they are named properly